### ISSUE TITLE
Unable to start the streaming using Nvidia Studio Driver 526.98

### ISSUE THREAD

**Issue by laolarou726** [moonlight-stream/moonlight-qt#903]
**Describe the bug**
Moonlight is stuck at the loading screen after the driver update to 526.98 (Studio Driver). The return error code is 4.

**Steps to reproduce**
- Try updating to the latest Studio Driver
- Launch the streaming
- Wait for the error

**Moonlight settings (please complete the following information)**
- Have any settings been adjusted from defaults?: **YES**
- If so, which settings have been changed?: **YES**
- Does the problem still occur after reverting settings back to default?: **YES**

**Gamepad-related issues (please complete if problem is gamepad-related)**
- Do you have any gamepads connected to your host PC directly?
- Does the problem remain if you stream the desktop and use https://html5gamepad.com to test your gamepad?
  - Instructions for streaming the desktop can be found here: https://github.com/moonlight-stream/moonlight-docs/wiki/Setup-Guide

**Client PC details (please complete the following information)**
 - OS: Windows 11 22H2
 - Moonlight Version: v4.3.1
 - GPU: Nvidia Geforce RTX 3080 10GB

**Server PC details (please complete the following information)**
 - OS: Windows 11 22H2
 - GeForce Experience version: 3.26.0.154
 - Nvidia GPU driver: Nvidia Studio Driver 526.98
 - Antivirus and firewall software: Windows Defender and Windows Firewall

**Moonlight Logs (please attach)**
[Moonlight-1669420623.log](https://github.com/moonlight-stream/moonlight-qt/files/10095352/Moonlight-1669420623.log)

**Comment by Angmar3019** [comment]
Hello, I ran into the same error myself.

But I think I have found the error. For me it always worked until I updated to the latest driver.

With the latest Nvidia driver (526.98) there is no longer the option "Shield" with the gamestream option in the settings. When I start Moonlight, I get the same error "Error 4" as you.
![Unbenannt](https://user-images.githubusercontent.com/86516888/204088782-0e267a4d-7034-4859-8055-925b3282d689.PNG)



**Workaround:**
Go to this [page](https://www.nvidia.com/download/find.aspx) from Nvidia and download an older version of your driver. I have installed the version 526.47. After the installation of the older driver there is also again the option with gamestream. Switch it on again and everything works normally.

I can not say why the option is no longer available in the latest driver, but I hope Nvidia does not remove this option forever.

**Comment by HimaCSR** [comment]
Hello, I ran into the same error myself.
But I have installed the version 526.47 ,even the 512.77 . It was not working . Is that Nvdia remove this option ?
Client PC details
OS: Windows 11 22H2
GPU: Nvidia Geforce RTX 3060 12GB
![image](https://user-images.githubusercontent.com/120147211/206606584-922486bb-b9c0-4f23-9e07-739c37294909.png)
![image](https://user-images.githubusercontent.com/120147211/206606744-ebcc11b8-df93-41a4-abbb-80f14d51bbb8.png)

**Comment by cgutman** [comment]
No, the option still exists. I'm running 527.56 (released today) and streaming still works fine.

When the Shield tab disappears, it usually means something in GeForce Experience or the driver broke. If a reboot doesn't solve it, I'd suggest a clean install of both.

If that also fails, you can report a bug here: https://nvidia.custhelp.com/app/ask
